core/access_api.php

```diff
@@ -895,3 +894,41 @@ function access_parse_array( array $p_access ) {
 
 	return $t_access_level;
 }
+
+/**
+ * Returns true if the user is limited for viewing issues, in the specified project,
+ * accordint to 'limit_reporters' options
```

Comment:
```
Returns true if the user has limited view to issues in the specified project.
```

Not sure what you mean by *accordint to 'limit_reporters' options*. Is this mention necessary ? If so, maybe better to xref it with `@see $g_limit_view_unless_threshold`